### ISSUE TITLE
Improve UI Design of Internship Steps Page

### ISSUE THREAD

**Issue by volkansahn** [kulkultech/open-source#43]
Internship Steps description seems so close to each other, it is a bit hard to follow steps. It could be great to improve traceability with some changes on UI design.

**Comment by empeje** [comment]
Good ideas, could you elaborate more about the aspect that we can greatly improve? Is it coming from the visual (UI) or more into the flow (UX)?

**Comment by volkansahn** [comment]
For UI side; it is hard to separate Step heads (like Step 0, Step 1, etc.) from context. May be we can put a separator before them or put them in a highlighted box. 

For UX side, I know it is a one page design but this page is long and navigation pane is not moving with you till the end of the page. For example if I scroll down to Step 9 and I want to go back to Step 0 I have 2 option; one is scroll up to Step 0, second is click Virtual Interns drop down list on top and click Step 0 which loads the page again. May be the the navigation pane can move with scroll, or if I click Step 0 from drop down list There could be a new page for Step 0.

**Comment by empeje** [comment]
For the UI, I think you already working on it, great job 🎉 

For the UX, I think that's a great idea, if you can move the pane with the scroll that would be very awesome.

**Comment by empeje** [comment]
Solved in #46
